build/FrameworkPackage/FrameworkPackageContents/AppxManifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:build="http://schemas.microsoft.com/developer/appx/2015/build" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" IgnorableNamespaces="uap build mp">
+  <Identity Name="$(PackageName)" ProcessorArchitecture="$(Platform)" Publisher="$(Publisher)" Version="$(Version)" />
+  <Properties>
+    <Framework>true</Framework>
+    <DisplayName>$(PackageName)</DisplayName>
+    <PublisherDisplayName>Microsoft Platform Extensions</PublisherDisplayName>
+    <Description>Microsoft Project Reunion</Description>
+    <Logo>logo.png</Logo>
+  </Properties>
+  <Resources>
+    <Resource Language="en-US" />
+  </Resources>
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.15063.0" MaxVersionTested="10.0.18362.0" />
+  </Dependencies>
+  <mp:PhoneIdentity PhoneProductId="200b7352-04ae-4693-8a97-23a38e7d67d4" PhonePublisherId="1AC97219-40D1-4c98-B175-2BFCD96578C3" />
+  <Extensions>
+$(ActivatableTypes)
+      <Extension Category="windows.activatableClass.inProcessServer">
+        <InProcessServer>
+          <Path>Microsoft.ProjectReunion.dll</Path>
+          <!-- Special type that we manually list here which is not part of the Nuget dll distribution package. 
+               This is our breadcrumb that we leave to be able to detect at runtime that we're using the framework package. -->
+          <!-- UNDONE <ActivatableClass ActivatableClassId="Microsoft.UI.Private.Controls.FrameworkPackageDetector" ThreadingModel="both" /> -->
```

Comment:
?

Please don't check in dead code

Reply:
(won't fix), I'm leaving this here for step 3, get the framework package working. Normally I wouldn't check in dead code but there's so much infrastructure here, I want to do this in parts.